templates/gaming/index.html

```diff
@@ -0,0 +1,209 @@
+{% extends '_layout.html' %}
+
+
+{% block title %}Stream Android games at scale in the cloud.{% endblock %}
+{% block meta_description %}A game streaming platform on Arm and x86. Deliver advanced gaming experiences to any device.
+{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1c9WcMRx1OlQxUntC6YF5oq_wjmwA5vI-RDZC926EYmo/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-half-and-half-reversed is-dark">
+  <div class="row u-equal-height u-vertically-center">
+  <div class="col-6">
+    <h1>
+    A game streaming platform on Arm and x86
+    </h1>
+    <p class="p-heading--four">
+    Deliver advanced gaming experiences to any device
+    </p>
+  </div>
+  <div class="col-6 u-align--center u-hide--small">
+    {{ image (
+    url="https://assets.ubuntu.com/v1/00caf669-Anbox_GameStream_Game.svg",
+    alt="",
+    width="152",
+    height="195",
+    hi_def=True,
+    loading="auto"
+    ) | safe
+    }}
+  </div>
+  </div>
+</section>
+
+<section>
+  <div class="p-strip u-no-padding--bottom">
+    <div class="row">
+      <div class="col-6">
+        <h4>
+          Streaming from the cloud is emerging as a channel for direct game distribution. However, the economics of game streaming solutions are challenging. Anbox Cloud is a highly scalable platform for game streaming with an attractive cost to performance ratio.
+        </h4>
+      </div>
+      <div class="col-6">
+        <p>
+          Video games were traditionally enjoyed on sophisticated game consoles or expensive desktop PCs. Increasingly, gaming experiences are becoming mobile, spontaneous and casual. This change is driven by the proliferation of smartphones, broadband internet access and the cloud.
+        </p>
+        <p>
+          With the cloud providing a high-performance and scalable computing backend for games, video streaming is emerging as an attractive distribution channel. Game streaming from the cloud does away with expensive devices. Gamers can play games hosted in the cloud on any smartphone, laptop or tablet with an identical experience as more expensive consoles.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="p-strip u-no-padding--top">
+    <div class="row u-align--center">
+      <div class="col-4 u-hide--small">
```

Comment:
You don't need `u-hide--small` on each column but move it up to the parent `p-strip`